How to get VisDrone labels for train?

## ❔Question
How to generate visdrone labels?

## Additional context

Transferred 356/362 items from yolov5s.pt
Scaled weight_decay = 0.0005
Optimizer groups: 62 .bias, 62 conv.weight, 59 other
train: Scanning '../VisDrone/VisDrone2019-DET-train/labels.cache' images and labels... 0 found, 6471 missing, 0 empty, 0 corrupted: 100%|█| 6471/6471 [00:00<?, ?it/Traceback (most recent call last):
  File "train.py", line 544, in <module>
    train(hyp, opt, device, tb_writer)
  File "train.py", line 189, in train
    dataloader, dataset = create_dataloader(train_path, imgsz, batch_size, gs, opt,
  File "/home/desmond/Projects/Python/yolov5/utils/datasets.py", line 63, in create_dataloader
    dataset = LoadImagesAndLabels(path, imgsz, batch_size,
  File "/home/desmond/Projects/Python/yolov5/utils/datasets.py", line 396, in __init__
    assert nf > 0 or not augment, f'{prefix}No labels in {cache_path}. Can not train without labels. See {help_url}'
AssertionError: train: No labels in ../VisDrone/VisDrone2019-DET-train/labels.cache. Can not train without labels. See https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data
train: Scanning '../VisDrone/VisDrone2019-DET-train/labels.cache' images and labels... 0 found, 6471 missing, 0 empty, 0 corrupted: 100%|█| 6471/6471 [00:00<?, ?it/
(env) desmond@Goodboy:~/Projects/Python/yolov5$

--- a/data/visdrone.yaml
+++ b/data/visdrone.yaml
@@ -56,7 +56,7 @@ download: |
   dir = Path('../VisDrone')  # dataset directory
   urls = ['https://github.com/ultralytics/yolov5/releases/download/v1.0/VisDrone2019-DET-train.zip',
           'https://github.com/ultralytics/yolov5/releases/download/v1.0/VisDrone2019-DET-val.zip',
-          'https://github.com/ultralytics/yolov5/releases/download/v1.0/VisDrone2019-DET-test-dev.zip'
+          'https://github.com/ultralytics/yolov5/releases/download/v1.0/VisDrone2019-DET-test-dev.zip',
           'https://github.com/ultralytics/yolov5/releases/download/v1.0/VisDrone2019-DET-test-challenge.zip']
   download(urls, dir=dir)
 

